### PR TITLE
Move null from Global Objects to Operators/Expressions

### LINKED ISSUE #16309: `null` is not a built-in object
### MDN URL

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/null

### What specific section or headline is this issue about?

_No response_

### What information was incorrect, unhelpful, or incomplete?

`null` is put under the "Built-in objects" category, which is incorrect, because `null` is a keyword, not a property.

### What did you expect to see?

I expect it to be moved to "Expressions & Operators", where `new.target`, `this`, etc. live.

That would separate this from `undefined`, which is desirable because `undefined` acts more like a "built-in object".

### Do you have any supporting links, references, or citations?

https://tc39.es/ecma262/#prod-ReservedWord

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/web/javascript/reference/global_objects/null`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/null
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/javascript/reference/global_objects/null/index.md
* Last commit: https://github.com/mdn/content/commit/edaa4aff4cf7532efdcba6014b9316aad2a5b6f1
* Document last modified: 2022-03-27T17:30:40.000Z

</details>

### PR DESCRIPTION
**Pull request by Josh-Cena**
#### Summary

Fix #16309

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/javascript/reference/global_objects/index.md [hunk 1]
```diff
@@ -163,7 +163,3 @@ Additions to the ECMAScript core for language-sensitive functionalities.
 - {{JSxRef("WebAssembly.CompileError")}}
 - {{JSxRef("WebAssembly.LinkError")}}
 - {{JSxRef("WebAssembly.RuntimeError")}}
-
-### Other
-
-- {{JSxRef("Functions/arguments", "arguments")}}
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/global_objects/index.md:169]: Orthogonal fix: I don't think this belongs.

File: files/en-us/web/javascript/reference/operators/null/index.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
 title: 'null'
-slug: Web/JavaScript/Reference/Global_Objects/null
+slug: Web/JavaScript/Reference/Operators/null
 tags:
   - JavaScript
   - Language feature
   - Literal
   - Primitive
-browser-compat: javascript.builtins.null
+browser-compat: javascript.operators.null
```
Comment (Josh-Cena) [inline files/en-us/web/javascript/reference/operators/null/index.md:9]: Pending BCD refactor

File: files/en-us/_wikihistory.json [hunk 3]
```diff
@@ -140074,7 +140074,7 @@
       "Anonymous"
     ]
   },
-  "Web/JavaScript/Reference/Global_Objects/null": {
+  "Web/JavaScript/Reference/Operators/null": {
```
Comment (Josh-Cena) [inline files/en-us/_wikihistory.json:140077]: Not familiar—is this supposed to be updated?

Comment (teoli2003) [reply files/en-us/_wikihistory.json:140077]: Yes, yari is adding redirects when we move pages. This is done automatically when you use `yarn content move <slug1> <slug2>` or `yarn content delete <slug1> --redirect <slug2>`.

Comment (Josh-Cena) [reply files/en-us/_wikihistory.json:140077]: Oh, TIL

**Review by sideshowbarker — APPROVED** [review]

**Pull request merged**